css/style.css

```diff
@@ -53,35 +54,159 @@ body {
     color: var(--main-bg-color)
 }
 
+h1{
+    font-family: 'Raleway';
+    font-size: 22px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+}
+
+h2 {
+    font-family: 'Raleway';
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+}
 /******Header ******/
+main{
+    padding-top: 30px;
+}
+
 nav {
     display: flex;
     justify-content: space-between;   
 }
 
-.nav_bar{
+.nav-bar{
     display: flex;
     align-items: center;
     justify-content: space-evenly;
     gap: 68px;
+    height: 50px;
 }
 
-.activitésButton:hover,
-.hebergementsButton:hover{
-    border-top: #0065FC solid 2px ;
-        
+.hebergementbutton:hover,
+.activitesbutton:hover{
+    border-top: #0065FC solid 2px;
+    color: #0065FC;    
+}
+
+
```

Comment:
Essayer d'ajouter des commentaires (/****** Hebergements And Populaires ***********/) pour separer les parties de css